README.md

```diff
@@ -103,8 +107,15 @@ for more detail on querying.
 
 ## Optional
 
-An alternative to using `curl` for ES config and querying is
-[kibana](https://www.elastic.co/products/kibana)
+An alternative to using `curl` for ES config, querying and also is visualisation [kibana](https://www.elastic.co/products/kibana)
+Use version [5.3.*](https://www.elastic.co/blog/kibana-5-3-1-released) that works with out ES image and follow the installation steps
```

Comment:
I wouldn't be specific about the version here as when the ES image gets updated this will be forgotten about and be the wrong version. I  suggest adding something about how the version of Kibana needs to match the version of ES. However, having said that, when starting Kibana, if the versions do not match a warning to that effect is output anyway.